renderer/reducers/neutrino.js

```diff
@@ -405,6 +435,26 @@ neutrinoSelectors.neutrinoSyncPercentage = createSelector(
   }
 )
 
+neutrinoSelectors.neutrinoRecoveryPercentage = createSelector(
+  blockHeightSelector,
+  neutrinoRecoveryHeightSelector,
+  neutrinoFirstRecoveryHeightSelector,
+  (blockHeight, neutrinoRecoveryHeight, neutrinoFirstRecoveryHeight) => {
+    // filters
+    const filtersToSync = blockHeight - neutrinoFirstRecoveryHeight
+    const filtersRemaining = blockHeight - neutrinoRecoveryHeight
+    const filtersDone = filtersToSync - filtersRemaining
+
+    const percentage = Math.floor((filtersDone / filtersToSync) * 100)
+
+    if (percentage === Infinity || Number.isNaN(percentage)) {
```

Comment:
Consider using `isFinite` to keep it more compact

Reply:
updated